src/auxinfo/participant.rs

```diff
@@ -83,6 +85,26 @@ pub enum Status {
 /// # 🔒 Storage requirements
 /// The [`AuxInfoPrivate`] output requires secure persistent storage.
 ///
+/// # High-level protocol description
+/// The auxinfo protocol runs in four rounds:
+/// - In the first round, each participant generates an RSA modulus `N = pq`,
+///   alongside ring-Pedersen parameters `(s, t, λ)` such that `s = t^λ mod N`.
+///   Each participant then produces a zero-knowledge proof `𝚷[prm]` that the
+///   ring-Pedersen parameters are correct. Finally, each participant commits to
+///   the tuple `(N, s, t, 𝚷[prm])` and broadcasts this commitment.
+/// - Once all participants have received all broadcasted commitments, in the
+///   second round each participant sends its own decommitment to all other
+///   participants.
+/// - In the third round, each participant (1) checks the validity of all the
+///   commitments plus the validity of the committed `𝚷[prm]` proof, and (2)
+///   produces two additional proofs: the first, `𝚷[mod]`, which asserts the
+///   validity of `N`, and the second, `𝚷[fac]`, which ????. Each participant
+///   sends these proofs to all other participants.
+/// - Finally, in the last round each participant checks the validity of the
+///   proofs from round three. If everything passes, each participant outputs
+///   the `(N, s, t)` tuples from all other parties, alongside its own secret
+///   primes `(p, q)`.
```

Comment:
Also its own `s, t`.

Reply:
I added working to say the `(N, s, t)` tuple is from all parties (including ourselves), which hopefully addresses this point.